packages/expo-cli/src/commands/client/index.ts

```diff
@@ -61,19 +61,19 @@ export default function (program: Command) {
         }
         if (!exp.ios) exp.ios = {};
 
-        if (!_.has(exp, 'facebookAppId') || !_.has(exp, 'facebookScheme')) {
+        if (!has(exp, 'facebookAppId') || !has(exp, 'facebookScheme')) {
```

Comment:
```suggestion
        if (!exp.facebookAppId || !exp.facebookScheme) {
```
Maybe something like this?